sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/DataGenerationIterator.java

```diff
@@ -50,18 +48,9 @@ public boolean hasNext() {
         final Map<Key, ObjectNode> newDocuments = _dataGenerator.generate(recordsToGenerate)
```

Comment:
Do we need this map ? And if yes then just pass it from _dataGenerator.generate() as we are not doing any filtering

Reply:
The size of that map is used in the line below

Reply:
Answering the question posed above here: We will not store documents for different entities into the same collection. The schema/format of the documents stored in a collection will be same. 

I am envisioning we'd have a separate workload, one that is not scheduled on a daily basis, to validate other scenarios e.g. composite indexes on a collection.